external/spring/README.md

```diff
@@ -0,0 +1,31 @@
+# External Spring Tests
+
+Third Party container tests help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue [#172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Spring tests are pulled from the [spring-test-suite](https://github.com/spring-projects/spring-boot.git) repository.
```

Comment:
Please change the first part of the sentence to be:
Spring tests are part of the external third-party application tests that help verify that the Adoptium binaries are good...

Reply:
I have updated the first part of the sentence as asked.